estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,42 @@
+from odoo import models, fields ,api, exceptions
+
+
+class EstatePropertyOffer(models.Model):
+    _name = "estate.property.offer"
+    _description = "Estate properties Offers"
```

Comment:
Better to have the description here as title case

```suggestion
    _description = "Estate Properties Offers"
```